Source/WebCore/CMakeLists.txt

```diff
@@ -2084,6 +2084,7 @@ set(WebCore_USER_AGENT_STYLE_SHEETS
     ${WEBCORE_DIR}/css/html.css
     ${WEBCORE_DIR}/css/htmlSwitchControl.css
     ${WEBCORE_DIR}/css/mathml.css
+    ${WEBCORE_DIR}/css/mathmlCore.css
```

Comment:
nit: I wonder if it should be called `mathmlCoreExtras.css`, since mathml.css actually has most of the CSS rules from MathML Core.